### PR TITLE
[Docs] Update application runtime documentation

### PR DESCRIPTION
**Pull request by rokatyy**
Jira - https://iguazio.atlassian.net/browse/ML-7552

### CODE REVIEW

**Review by liranbg — APPROVED** [review]

File: docs/runtimes/application.ipynb [hunk 1]
```diff
@@ -135,6 +135,40 @@
     ")"
```
Comment (alonmr) [inline docs/runtimes/application.ipynb:135]: <pre class="ql-syntax" spellcheck="false">Line #2.        # The name of the API gateway, defaults to -
</pre><p>the default name is used only when deploying with default api gateway. when you use <code>create_api_gateway</code> directly, you must provide a name </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6275/discussion/#comment-1743931310'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='8'/>

Comment (rokatyy) [reply docs/runtimes/application.ipynb:135]: Fixed in both places

File: docs/runtimes/application.ipynb [hunk 2]
```diff
@@ -135,6 +135,40 @@
...
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other API Gateway configurations\n",
```
Comment (jillnogold) [inline docs/runtimes/application.ipynb]: "### Other API gateway configurations\n",

File: docs/runtimes/application.ipynb [hunk 3]
```diff
@@ -135,6 +135,40 @@
...
+    "\n",
+    "There are also other parameters that can be configured when creating an API gateway:"
```
Comment (jillnogold) [inline docs/runtimes/application.ipynb]: "There are additional parameters that can be configured when creating an API gateway:"

File: docs/runtimes/application.ipynb [hunk 4]
```diff
@@ -135,6 +135,40 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "application.create_api_gateway(\n",
+    "    # The name of the API gateway\n",
+    "    name=\"my-api-gateway\",\n",
+    "    # Optional path of the API gateway, default value is \"/\". Note that given path should be supported by deployed application\n",
```
Comment (jillnogold) [inline docs/runtimes/application.ipynb:156]: # Optional path of the API gateway, default value is \"/\". The given path should be supported by the deployed application\n",

File: docs/runtimes/application.ipynb [hunk 5]
```diff
@@ -135,6 +135,40 @@
...
+    "    path=\"/\",\n",
+    "    # Set True to allow direct port access to the application sidecar\n",
```
Comment (jillnogold) [inline docs/runtimes/application.ipynb]: " # Set to True to allow direct port access to the application sidecar\n",

File: docs/runtimes/application.ipynb [hunk 6]
```diff
@@ -135,6 +135,40 @@
...
+    "    direct_port_access=False,\n",
+    "    # Set True to force SSL redirect, False to disable. Defaults to mlrun.mlconf.force_api_gateway_ssl_redirect()\n",
```
Comment (jillnogold) [inline docs/runtimes/application.ipynb]: " # Set t0 True to force SSL redirect, False to disable. Defaults to mlrun.mlconf.force_api_gateway_ssl_redirect()\n",

File: docs/runtimes/application.ipynb (hunk 4 above)
Comment (jillnogold) [inline docs/runtimes/application.ipynb]: " # Optional path of the API gateway, default value is \"/\". The given path should be supported by the deployed application\n",

File: mlrun/runtimes/nuclio/application/application.py [hunk 7]
```diff
@@ -436,8 +436,9 @@ def create_api_gateway(
         """
         Create the application API gateway. Once the application is deployed, the API gateway can be created.
         An application without an API gateway is not accessible.
-        :param name:                    The name of the API gateway, defaults to <function-name>-<function-tag>
-        :param path:                    Optional path of the API gateway, default value is "/"
+        :param name:                    The name of the API gateway
+        :param path:                    Optional path of the API gateway, default value is "/".
+            Note that given path should be supported by deployed application
```
Comment (jillnogold) [inline mlrun/runtimes/nuclio/application/application.py]: The given path should be supported by the deployed application

**Review by jillnogold — APPROVED** [review]

File: docs/runtimes/application.ipynb [hunk 8]
```diff
@@ -135,6 +135,40 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Other API gateway configurations\n",
+    "\n",
+    "There are additional parameters that can be configured when creating an API gateway::"
```
Comment (TomerShor) [inline docs/runtimes/application.ipynb]:
```suggestion
    "There are additional parameters that can be configured when creating an API gateway:"
```

**Review by TomerShor — CHANGES_REQUESTED** [review]
Realllyyy minor

File: docs/runtimes/application.ipynb (hunk 8 above)
Comment (rokatyy) [reply docs/runtimes/application.ipynb]: @TomerShor nice catch

**Review by TomerShor — APPROVED** [review]

**Pull request merged**